src/euphorie/client/browser/templates/panel-add-user-to-organisation.pt

```diff
@@ -59,6 +59,13 @@
                     />
                     <tal:i18n i18n:translate="">Manager/Policy maker</tal:i18n>
                   </label>
+                  <label>
+                    <input name="role"
+                           type="radio"
+                           value="consultant"
+                    />
+                    <tal:i18n i18n:translate="label_role_consultant">Consultant</tal:i18n>
```

Comment:
Is it wanted to have two labels for the same translation, see above

Reply:
It is not; well spotted.